src/RestaurantsContainer.test.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+
+import { render } from '@testing-library/react';
+
+import { useSelector } from 'react-redux';
+
+import given from 'given2';
+
+import restaurants from '../fixtures/restaurants';
+
+import RestaurantsContainer from './RestaurantsContainer';
+
+describe('RestaurantsContainer', () => {
+  const givenSelector = (data) => useSelector.mockImplementation((selector) => selector({
+    restaurants: data,
+  }));
```

Comment:
```suggestion
  useSelector.mockImplementation((selector) => selector({
    restaurants: given.restaurants,
  }));
```

given 은 위와 같이 간단하게 사용도 가능해요!

저렇게 따로 함수형태로 빼서 하지 않으려고 사용하는 거에요.